CMakeLists.txt

```diff
@@ -22,6 +22,10 @@ if (CMAKE_HOST_WIN32)
   set(NO_SYMLINKS_DEFAULT ON)
 endif()
 
+if (CMAKE_GENERATOR MATCHES "Visual Studio")
+  message(SEND_ERROR "Visual Studio generators are not supported. Please use Ninja (-GNinja) instead.")
```

Comment:
This is not correct. you can use Visual Studio generators with clang-cl like so:

```
cmake -B build -T ClangCL
```

Reply:
That problem is fixed by my open PR to serenity in Assertions.h